src/backend/database/seeders/TicketSeeder.php

```diff
@@ -186,7 +186,7 @@ private function generateTicket() {
         ]);
 
         if ($randomCategory->out_source) {
-            $ticket->assigned_id = null;
+            $ticket->assigned_id = null; // TODO: why was this set to null?
```

Comment:
@dmohns  Any reason we set the `assigned_id` to null for `out_source` tickets?

Reply:
🤔 Can't remember why this was set to `null`. But feel free to change it, if it works for Ticket data seeding 👌